dashboard/app/models/hint_view_request.rb

```diff
@@ -0,0 +1,43 @@
+# == Schema Information
+#
+# Table name: hint_view_requests
+#
+#  id            :integer          not null, primary key
+#  user_id       :integer
+#  script_id     :integer
+#  level_id      :integer
+#  feedback_type :integer
+#  feedback_xml  :text(65535)
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
```

Comment:
Not knowing what I'm talking about... Many of the other tables do not have created_at and updated_at as NOT NULL. In particular, I think we tend to create a row and populate the row as separate operations for some tables. In between those operations, I imagine updated_at might be null. Please confirm with someone that knows more than I do on this.